src/qs.ts

```diff
@@ -7,7 +7,7 @@
  * file that was distributed with this source code.
  */
 
-import { parse, stringify } from 'qs'
+import { parse, stringify } from 'neoqs'
```

Comment:
If this package still ships CommonJS in any capacity, this should be `neoqs/legacy`. Same for the other file